packages/ember/tests/routing/basic_test.js

```diff
@@ -2127,46 +2107,6 @@ test('The rootURL is passed properly to the location implementation', function()
   bootApplication();
 });
 
-
-test('Only use route rendered into main outlet for default into property on child', function() {
```

Comment:
This one doesn't seem related to templateName, you can likely remove the templateName usage below and allow menu/view template be resolved.